.circleci/test_docker_container.sh

```diff
@@ -2,6 +2,12 @@
 
 set -xeuo pipefail
 
+# test for working en_US.UTF-8 locale
+locale -a | grep -i 'en_US.UTF.\?8'
+[ "$( LC_ALL=en_US.UTF-8 sh -euc : 2>&1 )" = "" ]
```

Comment:
This would spit out
```
/usr/bin/sh: warning: setlocale: LC_ALL: cannot change locale (en_US.UTF-8): No such file or directory
```
on `stderr` if the locale wouldn't work. If it works, we just get no output and can test for that.